src/answerfile.cpp

```diff
@@ -169,6 +169,19 @@ void AnswerFile::loadHostnameSettings()
         = m_ini.getValue("hostname", "domain_name", false, false);
 }
 
+Cluster::SELinuxMode AnswerFile::checkSELinuxMode(const std::string& mode)
+{
+    if (mode == "enforcing") {
+        return Cluster::SELinuxMode::Enforcing;
+    } else if (mode == "permissive") {
+        return Cluster::SELinuxMode::Permissive;
+    } else if (mode == "disabled") {
```

Comment:
We should have a method to ignore case when comparing strings. It will avoid bugs where the user selects `Disabled` and `disabled`. That should not be an issue in a TUI interface, but on an answerfile it is.